### PR TITLE
feat: config for html title

### PR DESCRIPTION
**Pull request by link-duan**
**Describe the PR**
config for html title

### CODE REVIEW

File: swagger.go [hunk 1]
```diff
@@ -194,7 +197,7 @@ const swagger_index_templ = `<!-- HTML for static distribution bundle build -->
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Swagger UI</title>
+  <title>{{.Title}}</title>
```
Comment (ubogdan) [inline swagger.go:200]: For backward compatibility please put the title in the default config.

**Review by ubogdan — CHANGES_REQUESTED** [review]

**Comment by ubogdan** [comment]
I guess there is no open issue for this.
Before you follow the requested changes please describe your use case so we can better understand the need.

File: swagger.go (hunk 1 above)
Comment (link-duan) [reply swagger.go:204]: supported

**Review by ubogdan — APPROVED** [review]
LGTM

**Comment by ubogdan** [comment]
Can you please give us some insights regarding the use case?

**Comment by link-duan** [comment]
> Can you please give us some insights regarding the use case?

The title of swagger web page (/swagger/index.html) is "Swagger UI". 
So if we open multiple "Swagger UI" pages, its hard to find the one we want from browser tabs.
So I think it's a good idea to support the customization of page title.

**Pull request merged**